mysql-test/lib/mtr_report.pm

```diff
@@ -576,6 +587,17 @@ sub mtr_print_line () {
   print '-' x 74 . "\n";
 }
 
+sub replace_special_symbols($) {
+  my $text= shift;
+  $text =~ s/[\"]//g;
+  $text =~ s/&/&#38;/g;
+  $text =~ s/'/&#39;/g;
+  $text =~ s/"/&#34;/g;
+  $text =~ s/</&lt;/g;
```

Comment:
you first you remove double quotes, then you replace them? Looks rather redundant :smile: 

remove the first `s/[\"]//g`

Reply:
Deleted it.